docs/searchqueryset_api.rst

```diff
@@ -342,6 +342,28 @@ Example::
     # Count document hits for authors that start with 'jo' within the index.
     SearchQuerySet().filter(content='foo').query_facet('author', 'jo*')
 
+``range_facet``
```

Comment:
Please make a note of the field types that are applicable - also add to the query docs. Per solr docs: "one can use the Range Faceting feature on any date field or any numeric field that supports range queries"